ibc/docs/README.md

```diff
@@ -0,0 +1,88 @@
+# IBC <!-- omit in toc -->
+
+- [Overview](#overview)
+- [IBC Module](#ibc-module)
+  - [Node Configuration](#node-configuration)
+- [Components](#components)
+  - [ICS-24 Host Requirements](#ics-24-host-requirements)
+
```

Comment:
Wdyt about adding a persistence (or something) section which covers what network state this module interacts with?

Reply:
I have added a brief section to cover this but the rest of the persistence info will come in the ICS-24 Store PR as this is where interfacing with persistence actually comes into play